plugins/module_utils/occ.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2022, Marc Crébassa <aalaesar@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+import os
+# import json
+
+
+def run_occ(module, command, ):
```

Comment:
Formatting of this file is very odd. Try to run black in whole folder, also useful to have on pre-commit.

[.pre-commit-config.yaml](https://github.com/psf/black/blob/main/.pre-commit-config.yaml):

```
---
repos:
  - repo: https://github.com/psf/black
    rev: 22.12.0
    hooks:
      - id: black
        language_version: python3.11
```

Reply:
just done it.
how about now ? 😃